custom-words.txt

```diff
@@ -1526,6 +1526,7 @@ polyline
 poolusagemetrics
 popreceipt
 portalsettings
```

Comment:
Adding a comment to block this PR until we figure some things out internally at APIM.